package.json

```
@@ -34,6 +46,19 @@
     "clean": "gatsby clean",
     "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "jest",
+      "post-push": "npm run build"
+    }
+  },
+  "lint-staged": {
+    "./src/*.{js,jsx,ts,tsx}": [
+      "npx prettier --write",
+      "eslint src/*.js --fix-dry-run"
```

La configuration del eslint esta erronea, solo estaremos utilizando typescript en el proyecto.
```suggestion
      "eslint . --fix-dry-run"
```